devito/profiler.py

```diff
@@ -81,18 +86,23 @@ def add_profiling(self, code, name, byte_size=4, omp_flag=None):
                           Defaults to 4.
         :param omp_flag: OpenMP flag to add before profiling operations,
                          if needed.
+        :param to_ignore: List of strings containing the labels of
+                          symbols used as loop variables
```

Comment:
Hard to say, but we'll need to either re-invent or retro-fit almost all of the `Propagator` internals to the `Iteration`/`Expression` classes, including pragmas, loop blocking, etc. I'll update issue #43, and get on it as soon as I can. In the meantime, I think the `to_ignore` approach is a good enough placeholder, so it shouldn't stop this merge.